llvm/lib/Target/X86/X86TargetTransformInfo.cpp

```diff
@@ -6756,3 +6756,8 @@ InstructionCost X86TTIImpl::getScalingFactorCost(Type *Ty, GlobalValue *BaseGV,
     return AM.Scale != 0;
   return -1;
 }
+
+InstructionCost X86TTIImpl::getBranchMispredictPenalty() const {
+  // TODO: Hook MispredictPenalty of SchedMachineModel into this.
+  return 14;
```

Comment:
How did you arrive at the value 14? Why is the TODO a TODO and not implemented in this PR?